sku.routes.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const path = require('path');
+const partition = require('lodash/partition');
+
+const componentsPath = path.join(__dirname, 'lib/components/index.js');
+const componentsSource = fs.readFileSync(componentsPath, 'utf-8'); // eslint-disable-line no-sync
+const components = componentsSource
+  .match(/default as [A-Za-z]+/g)
```

Comment:
Using a positive lookbehind regex should make the map unnecessary, coupled with a positive lookahead will allow us to support component names with non a-z chars in the future (maybe Heading.Level1?):
```
/(?<=\{ default as ).*(?= \})/g
```